app/cafe.py

```diff
@@ -0,0 +1,23 @@
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated.")
+
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Vaccine has expired.")
+
+        if not visitor["wearing_a_mask"]:
```

Comment:
would be better to use .get method here in case if this key is not in dictionary

Reply:
please change

Reply:
I've already changed it. Maybe you opened the wrong commit?

Reply:
No you haven't